examples/svm.py

```diff
@@ -110,3 +110,10 @@ def svm_from_cfg(cfg):
 inc_value = svm_from_cfg(incumbent)
 
 print("Optimized Value: %.2f" % (inc_value))
+
+
+# We can also validate our results (though this makes a lot more sense with instances)
```

Comment:
I thought of the second one. We could have the RF example be the rather simple use case without categorical hyperparameters and instances and the SVM be the complicated one showcasing what SMAC can do as well.